cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
         )
```

Comment:
The `validate` method in `TicketSerializer` calls `Ticket.validate_ticket` with `attrs` values. Ensure that `attrs` contains the keys 'row', 'seat', and 'movie_session' before accessing them to avoid potential `KeyError`. Consider adding a check or using `attrs.get('key')` with a default value.